src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/CreateCustomPipelineAction.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2006-2020 North Concepts Inc.  All rights reserved.
+ * Proprietary and Confidential.  Use is subject to license terms.
+ * 
+ * https://northconcepts.com/data-pipeline/licensing/
+ */
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.northconcepts.datapipeline.core.ArrayValue;
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.Record;
+import com.northconcepts.datapipeline.foundations.file.LocalFile;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.action.PipelineAction;
+import com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.output.ExcelPipelineOutput;
+import com.northconcepts.datapipeline.foundations.sourcecode.CodeWriter;
+import com.northconcepts.datapipeline.foundations.sourcecode.JavaCodeBuilder;
+import com.northconcepts.datapipeline.internal.lang.Util;
+import com.northconcepts.datapipeline.transform.BasicFieldTransformer;
+import com.northconcepts.datapipeline.transform.TransformingReader;
+
+public class CreateCustomPipelineAction {
+
+    public static void main(String[] args) {
+        CsvPipelineInput pipelineInput = new CsvPipelineInput()
+                .setFileSource(new LocalFile().setPath("example/data/input/countries_with_country-code.csv"))
+                .setFieldNamesInFirstRow(true);
+
+        ExcelPipelineOutput pipelineOutput = new ExcelPipelineOutput()
+                .setFileSink(new LocalFile().setPath("example/data/output/countries_with_country-code.xlsx"))
+                .setFieldNamesInFirstRow(true);
+
+        Pipeline pipeline = new Pipeline()
+                .setInput(pipelineInput)
+                .setOutput(pipelineOutput)
+                .addAction(new UpcaseFieldsAction().addField("Country"));
+
+        pipeline.run();
+
+        System.out.println("---------------------------------------------------------------------------------------------------------");
+
+        System.out.println("Generated Code:" + System.lineSeparator());
```

Comment:
Can we drop the `System.lineSeparator()` call (here and below)?  Are they needed?

Reply:
Removed all `System.lineSeparator()`.